packages/strapi-admin/ee/admin/containers/Roles/CreatePage/ContentTypeCollapses/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { memo, useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import ContentTypeCollapse from '../ContentTypeCollapse';
+
+const ContentTypeCollapses = ({ actions, subjects }) => {
+  const [collapseToOpen, setCollapseToOpen] = useState(null);
+
+  const handleClickToggleCollapse = useCallback(
```

Comment:
Is `useCallback necessary here?`

ContentTypeCollapse is not memorized and collapseToOpen is bound to the state component: it will render everytime it changes.

Reply:
You are right! I was hoping to create a `useCollapse` hook so I anticipated the implementation a bit and I forgot to develop it.

Reply:
Fixed in #9405